docs/_basic/ja_authenticating_oauth.md

```diff
@@ -50,6 +50,18 @@ const app = new App({
       }
       throw new Error('Failed fetching installation');
     },
+    deleteInstallation: async (installQuery) => {
+      // change the line below so it deletes from your database
```

Comment:
@seratch Could you assist with the appropriate translation in this section?

Reply:
```suggestion
      // 実際のデータベースから削除するために、ここのコードを変更
```